android-pay/src/main/java/com/stripe/wrap/pay/activity/StripeAndroidPayActivity.java

```diff
@@ -567,9 +565,6 @@ protected void onChangedMaskedWalletRetrieved(@Nullable MaskedWallet maskedWalle
      */
     @CallSuper
     protected void onMaskedWalletRetrieved(@Nullable MaskedWallet maskedWallet) {
-        if (mBuyButtonFragment != null) {
```

Comment:
Worth noting: this removal is because I realized it was already being done right before this method was called in onActivityResult